zlink-core/src/server/service.rs

```diff
@@ -43,7 +43,7 @@ pub trait Service {
     /// Handle a method call.
     fn handle<'ser, 'de: 'ser, Sock: Socket>(
         &'ser mut self,
-        method: Call<Self::MethodCall<'de>>,
+        method: &'ser Call<Self::MethodCall<'de>>,
```

Comment:
Hmm.. shouldn't it become:

```suggestion
        method: &'de Call<Self::MethodCall<'_>>,
```

🤔

Reply:
Hm, you're right. Not sure what made me go for `'ser` there. I think we can even reduce it down to just a single lifetime again (since the anonymous lifetime for `MethodCall<'_>` is now constrained by the reference itself).

Reply:
yeah, LGTM. I just realized, isn't this an API break? `cargo semver` (that I recently added to the CI) doesn't seem to catch it. :( I guess we need to file an issue for it. I was hoping that next releases won't be breaking but this seems like a good reason to do it. We'll have to bump the semver version though.

Reply:
> it doesn't detect the change in the number of lifetime parameters for the method.

that's a bit understandable, given how lifetimes can be quite confusing for people. :) However, here we're changing the argument from owned to reference type so there are 2 breakages it's missing on.

Anyway, please bump the semver.